themes/digital.gov/src/scss/new/_services-directory.scss

```diff
@@ -6,7 +6,7 @@
     @include u-margin-bottom(5);
   }
 }
-.card-service-contact-heading{
+.service-contact-heading{
   @include u-display('none');
   @include at-media('tablet') {
     @include u-display('block');
```

Comment:
Let's run prettier formatting on this partial to use consistent quotes and add spacing before `{`.